### PR TITLE
Some information about the Zurich and the venue

### PR DESCRIPTION
**Pull request by gandro**
Could use some more polish, but most information is there.

### CODE REVIEW

File: location.md [hunk 1]
```diff
@@ -4,49 +4,68 @@ title: "Next stop: Zürich"
 permalink: /location/
 ---
 
-<div class="backdrop" style="background: url(/assets/backdrops/zurich.jpg) 50% 80%; padding: 0; background">
+<div class="backdrop" style="background: url(/assets/zurich/30615001992_50cdc23e7b_k.jpg) 50% 40%; padding: 0; background">
```
Comment (Hoverbear) [inline location.md]: Please use more semantic filenames. :)

File: location.md [hunk 2]
```diff
@@ -4,49 +4,68 @@ title: "Next stop: Zürich"
...
+
+<p>While it is probably best known for being a financial hub, Zurich is also home to many technology and research endeavors: With three state universities, many tech companies ranging from small startups to large international enterprises, Zurich is a great place for developers and enthusiasts to be in.</p>
 </section>
 
 <section class="img-grid">
 
 <div>
-<img src="/assets/zurich/31221045461_e756a87f90_z.jpg">
-<img src="/assets/zurich/4567067975_70a6129cde_o.jpg">
+<img src="/assets/zurich/30615001992_50cdc23e7b_k.jpg">
```
Comment (Hoverbear) [inline location.md]: Again file names.

File: location.md [hunk 3]
```diff
@@ -4,49 +4,68 @@ title: "Next stop: Zürich"
...
-<div class="img-grid">
-<img src="/assets/zurich/venue-inside.jpg" alt="Foyer of venue at ETH">
-</div>
+### Getting there
+
+The address of the venue is *[Sonneggstrasse 5, 8092 Zurich](https://goo.gl/maps/K5Zb1P8ARdE2)*.
 
+Zurich has excellent public transport - the venue can easily be reached by tram no. 6, 10 and 9 by exiting the stop "ETH/Universitätsspital". Coming from Zurich Main Station ("Zurich HB"), there is also the option to take the Polybahn funicular from "Central" to "ETH Zentrum". Look out for the *focusTerra* signs to find the NO building.
+
+A 24 hour ticket for within the city of Zurich (zone 110) costs CHF 8.80, a single tick 4.40 CHF. Traveling from the airport, you will require a ticket valid for 3 zones (single ticket 6.80 CHF).
```
Comment (Hoverbear) [inline location.md]: `tick` -> `ticket`

File: location.md [hunk 4]
```diff
@@ -4,49 +4,68 @@ title: "Next stop: Zürich"
...
+<img src="/assets/zurich/eth-mainbuilding.jpg" alt="Main building of ETH by night">
+<img src="/assets/zurich/venue-inside.jpg" alt="Foyer of the NO building">
+</div>
 </section>
 
 <hr>
 
 <section style="font-size: 0.65em">
-    <p><strong>Credits:</strong> https://www.flickr.com/photos/stephen-oung/sets/72157625972849555 https://www.flickr.com/photos/lschlagenhauf/sets/72157664912678546</p>
+    <p><strong>Image Credits:</strong> <a href="https://www.flickr.com/photos/lschlagenhauf/30615001992/in/album-72157664912678546/">Zurich purple hour</a> by
```
Comment (Hoverbear) [inline location.md:68]: 👍

**Comment by gandro** [comment]
Fixed!

**Review by Hoverbear — APPROVED** [review]

**Pull request merged**